doc/classes/TouchScreenButton.xml

```diff
@@ -30,9 +30,7 @@
 			The button's texture for the normal state.
 		</member>
 		<member name="passby_press" type="bool" setter="set_passby_press" getter="is_passby_press_enabled" default="false">
-			If [code]true[/code], the [code]pressed[/code] and [code]released[/code] signals are emitted
-			whenever a pressed finger goes in and out of the button,
-			even if the pressure started outside the active area of the button.
+			If [code]true[/code], the [code]pressed[/code] and [code]released[/code] signals are emitted whenever a pressed finger goes in and out of the button, even if the pressure started outside the active area of the button.
```

Comment:
Use `[signal pressed]` and `[signal released]` to make hyperlinks to signals instead of using inline code markup. See https://docs.godotengine.org/en/latest/community/contributing/class_reference_writing_guidelines.html#improve-formatting-with-bbcode-style-tags

Please also squash the commits into one.

Reply:
Note: The CI failures are unrelated to this PR, should be fixed with the next push.

Reply:
done